pay-api/src/pay_api/models/invoice.py

```diff
@@ -352,6 +356,7 @@ def from_row(
             None if row.business_identifier and row.business_identifier.startswith("T") else row.business_identifier
         )
         line_items = [PaymentLineItemSearchModel.from_row(x) for x in row.payment_line_items]
+        partial_refunds = [RefundsPartialSearchModel.from_row(x) for x in row.partial_refunds]
```

Comment:
This is going to change the payload probably for CSO which is going to break their code